app/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Step 0 - Build
+FROM node:10 AS build
+
+# If REACT_APP_HTTP_URI variable is not passed as argument during image build,
+# it defaults to "http://localhost:4000"
+# i.e. docker build --build-arg REACT_APP_HTTP_URI=${ANY REACT_APP_HTTP_URI} ...
+ARG REACT_APP_HTTP_URI="http://localhost:4000"
+ENV REACT_APP_HTTP_URI=$REACT_APP_HTTP_URI
+
+COPY . .
+RUN yarn global add serve
```

Comment:
I wasn't able to solve the issue that was popping with Cloud Run regarding the "missing error.log file". I went through all Stackoverflow to try and find something. I did see some people discussing about having similar issues but none with any resolution.

I did as well an official [GCP Nginx setup for Cloud Run](https://cloud.google.com/community/tutorials/deploy-react-nginx-cloud-run) and that didn't work as well.

And basically all the hassle and complications forced me to look for alternatives. [Serve](https://www.npmjs.com/package/serve) is a very simple web server for serving static files. I like it because it's pretty straight forward and gets the job done. Plus it worked great with Cloud Run the very first time I tried.